docs/manage_infrastructure.md

```diff
@@ -425,6 +425,32 @@ It works even if the VM is running, because we'll automatically export a snapsho
 
 In the VM "Snapshots" tab, you can also export a snapshot like you export a VM.
 
+## VM migration
+
+### Simple VM Migration (VM.pool_migrate)
+
+Simple migration involves moving the VM without moving its disks. This feature is only possible when the VM's disks are on a shared SR by both hosts.
+
+#### Use Case
+
+- Move a VM within the same pool from host A to host B without moving the VM's VDIs.
```

Comment:
"Move" is a bit ambiguous here. I guess the main use case is to take a VM that is running on host A and run it on host B simply by transferring its RAM content from A to B. Or is there also a use case for halted VMs? Is it even possible to use this on halted VMs?